models/issue.go

```diff
@@ -1210,7 +1210,8 @@ func (opts *IssuesOptions) setupSession(sess *xorm.Session) {
 	}
 
 	if opts.IsArchived != util.OptionalBoolNone {
-		sess.Join("INNER", "repository", "issue.repo_id = repository.id").And(builder.Eq{"repository.is_archived": opts.IsArchived.IsTrue()})
+		sess.Join("INNER", "repository", "`issue`.repo_id = `repository`.id").
```

Comment:
When the method is called with opts.IsArchived = util.OptionalBoolNone, won't the absence of the join cause a problem in GetRepoIDsForIssuesOptions()?

Reply:
Fixed